src/renderer/helpers/rx/liveData.ts

```diff
@@ -12,6 +12,7 @@ import { Filterable2 } from 'fp-ts/lib/Filterable'
 import { MonadThrow2 } from 'fp-ts/lib/MonadThrow'
 import { pipeable } from 'fp-ts/lib/pipeable'
 import { Observable } from 'rxjs'
+import { map as mapStream } from 'rxjs/operators'
```

Comment:
Could you use same naming convention as we do in other modules for import of `'rxjs/operators'?
```
import * as RxOp from 'rxjs/operators'
```